hkm/views/views.py

```diff
@@ -1096,23 +1091,35 @@ def handle_download(self, request, *args, **kwargs):
                   'data': {'url': tmp_image.edited_image.url}})
         return http.JsonResponse({'url': tmp_image.edited_image.url})
 
+
+class AjaxAddToCollection(View):
+    name = 'hkm_add_to_collection'
+
+    def post(self, request, *args, **kwargs):
+        if request.user.is_authenticated():
+            action = request.POST['action']
+            if action == 'add':
+                return self.handle_add_to_collection(request, *args, **kwargs)
+            elif action == 'add-create-collection':
+                return self.handle_add_to_new_collection(request, *args, **kwargs)
+        return http.HttpResponseForbidden()
```

Comment:
Now also incorrect actions will get this 403 which is a bit off. Not a huge deal in this exact spot ofc.

Reply:
Changed this back to `http.HttpResponseBadRequest()`